Sources/Commands/PackageCommands/EditCommands.swift

```diff
@@ -32,16 +32,16 @@ extension SwiftPackageCommand {
         @Option(help: "Create or use the checkout at this path")
         var path: AbsolutePath?
 
-        @Argument(help: "The name of the package to edit")
-        var packageName: String
+        @Argument(help: "The identity of the package to edit.")
```

Comment:
Nit: none of the other help strings have a period

Reply:
Good catch, thank you!